### PR TITLE
Add FIAM snippets

### PR DESCRIPTION
**Pull request by samtstern**

### CODE REVIEW

File: inappmessaging/FIAMReference/FIAMReference/AppDelegate.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  FIAMReference
+//
+//  Created by Sam Stern on 5/20/19.
+//  Copyright © 2019 Google. All rights reserved.
+//
```
Comment (ulukaya) [inline inappmessaging/FIAMReference/FIAMReference/AppDelegate.h]: please update all headers with Apache 2.0

File: inappmessaging/FIAMReference/FIAMReference.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,664 @@
...
+		};
+		D729F054533B6E9D0CE26935 /* [CP] Embed Pods Frameworks */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${PODS_ROOT}/Target Support Files/Pods-FIAMReference/Pods-FIAMReference-frameworks.sh",
```
Comment (ulukaya) [inline inappmessaging/FIAMReference/FIAMReference.xcodeproj/project.pbxproj]: please run "pod deintegrate" and search for "pod" in the file. There shouldn't be any occurrence of word "pod"

File: inappmessaging/FIAMReference/FIAMReference/AppDelegate.m [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    return YES;
+}
+
+
+- (void)applicationWillResignActive:(UIApplication *)application {
```
Comment (ulukaya) [inline inappmessaging/FIAMReference/FIAMReference/AppDelegate.m]: please delete all empty functions.

File: inappmessaging/FIAMReference/FIAMReference/AppDelegate.m [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    return YES;
```
Comment (ulukaya) [inline inappmessaging/FIAMReference/FIAMReference/AppDelegate.m]: missing [FIRApp configure] ?

File: inappmessaging/FIAMReference/FIAMReferenceSwift/AppDelegate.swift [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        // [START fiam_register_delegate]
+        // Register the delegate with the InAppMessaging instance
+        let myFiamDelegate = CardActionFiamDelegate()
+        InAppMessaging.inAppMessaging().delegate = myFiamDelegate;
+        // [END fiam_register_delegate]
+        
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {}
```
Comment (ulukaya) [inline inappmessaging/FIAMReference/FIAMReferenceSwift/AppDelegate.swift]: delete all empty functions

**Comment by samtstern** [comment]
cc @markarndt

File: storage/StorageReference.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -9,18 +9,26 @@
 /* Begin PBXBuildFile section */
 		1090B0951EBCE0E200A8F759 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1090B0941EBCE0E200A8F759 /* AppDelegate.swift */; };
 		1090B0971EBCE0EE00A8F759 /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1090B0961EBCE0EE00A8F759 /* ViewController.swift */; };
+		1C785C22FFEEC5D370FD083E /* Pods_StorageReferenceSwift.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 3B25D74411D4C1ED684CA0A1 /* Pods_StorageReferenceSwift.framework */; };
```
Comment (ulukaya) [inline storage/StorageReference.xcodeproj/project.pbxproj]: run "pod deintegrate" 
delete all lines includes word "pod"

**Pull request merged**